infra/common/src/test/java/cn/hippo4j/common/extension/design/AbstractSubjectCenterTest.java

```diff
@@ -0,0 +1,47 @@
+package cn.hippo4j.common.extension.design;
+
+import lombok.Getter;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.util.concurrent.atomic.AtomicInteger;
+
+public class AbstractSubjectCenterTest {
```

Comment:
Unit test is too simple, please cover every method.

Reply:
Methods should be covered by multiple unit test cases instead of one unit test covering all methods.

Reply:
ok